framework/PostProcessors/MCSimporter.py

```diff
@@ -129,34 +129,7 @@ def run(self, inputs):
     if beFileFound==False and self.expand==True:
       self.raiseAnError(IOError, 'MCSImporterPostProcessor Post-Processor ' + self.name + ', Expand is set to False but no file with type=BElist has been found')
 
-    '''
-    self.mcsList=[]
-    self.beList=set()
-    self.probability = np.zeros((0))
-    self.mcsIDs = np.zeros((0))
-
-    # construct the list of MCSs and the list of BE
-    counter=0
-    with open(MCSlistFile.getFilename(), 'r') as file:
-      next(file) # skip header
-      lines = file.read().splitlines()
-      for l in lines:
-        elementsList = l.split(',')
-
-        self.mcsIDs=np.append(self.mcsIDs,elementsList[0])
-        elementsList.pop(0)
-
-        self.probability=np.append(self.probability,elementsList[0])
-        elementsList.pop(0)
-
-        for element in elementsList:
-          element.rstrip('\n')
-        self.mcsList.append(elementsList)
-        counter = counter+1
-        if self.expand==False:
-          self.beList.update(elementsList)
-    '''
-    self.mcsIDs, self.probability, self.mcsList, self.beList = MCSreader(MCSlistFile)
+    self.mcsIDs, self.probability, self.mcsList, self.beList = mcsReader(MCSlistFile)
```

Comment:
camelBack for ``MCSlistFile``, maybe ``mcsListFile``

Reply:
changed into mcsListFile